lib/rubocop/cop/style/guard_clause.rb

```diff
@@ -134,14 +134,12 @@ def consecutive_conditionals?(parent, node)
           end
         end
 
-        def opposite_keyword(node)
-          node.if? ? 'unless' : 'if'
-        end
```

Comment:
This is not strictly related to my change but I noticed it could be replaced with `node.inverse_keyword`.

https://github.com/rubocop/rubocop-ast/blob/816dfe7f2ca4e92c7eda226a9e8b44aa9fa81e81/lib/rubocop/ast/node/if_node.rb#L61-L73